exercises/concept/making-the-grade/.docs/introduction.md

```diff
@@ -1,96 +1,71 @@
-As soon as this expression evaluates to false, no more iterations of the loop will run.
-- The **expression** component is some code that will run at the end of each iteration.
+#include<array>
+#include <string>
 
-
-The `while loop` executes its body as long as its condition check is true.
-The code snippet below shows how to transform a for into a while loop.
-
-```cpp
-init-statement;
-while(condition) {
-    some_statement;
-    expression;
-}
+// std::array variable_name<element_type, size> {list of elements}
+std::array<std::string, 3> indie_rock {"yeah", "yeah", "yeah"};
+// indie_rock contains the elements "yeah" three times
 ```
 
-### Interlude: Increments and Decrements
-
-When working with loops is often required to add 1 or subtract 1 from a counter variable.
-This is so common, that the operation has special operators: `++` and `--`.
-
-They come in a prefix and a postfix form.
-The prefix changes the variable before use in the statement and the postfix version afterward.
-You probably want the prefix version most of the time.
+Vectors usually need more space, as they allocate memory for further growth.
+You do not need to specify a size.
 
 ```cpp
-int a{3};
-int b{--a};
-// b is 2, a is now 2
-int c{a++};
-// c is 2, a is now 3
+#include<vector>
+
+// std::vector variable_name<element_type> {list of elements}
+std::vector countdown<int> {3, 2, 1};
+// my_vector contains the elements 3, 2 and 1
 ```
 
-### For Loops - An example
+> Note: You do not need to know the exact mechanics behind the template concept yet, it will have its own concept further down the syllabus tree.
 
-The init component usually sets up a counter variable, the condition checks whether the loop should be continued or stopped and the post component usually increments the counter at the end of each repetition.
+### Element access
+
+Vectors and arrays share the same functions to access their elements.
+You can use the member functions `front` and `back` to get the first and last elements of the container.
+There is also `at` and the `[]` operator to access speficic elements.
 
 ```cpp
-int sum{0}
-for (int i{1}; i < 10; ++i) {
-  sum += i;
-}
+countdown[0];
+// => 3
+countdown.at(2);
+// => 1
+countdown[1] = 4;
+// countdown now contains 3, 4 and 1
+indie_rock.back() = "yeahs"; 
+// indie_rock is now "yeah", "yeah", "yeahs"
 ```
 
-This loop will sum the numbers from `1` to `9` (including `9`). 
+> Note: If you pick a position that is not in the range of your container, `[]` might lead to undefined behavior.
+> The `at` function would raise an exception, which might make your life easier in the long term.
 
-### Break and Continue
+### Vector modifiers
 
-Inside a loop body, you can use the `break` keyword to stop the execution of the loop entirely:
+As the vector is not fixed in size, it is possible to add or remove elements. Two common functions for that purpose are `emblace_back` and `pop_back`.
```

Comment:
```suggestion
As a `vector` is not fixed in size, it is possible to add or remove elements. 
Two common functions for that purpose are `emplace_back` and `pop_back`.
```

Reply:
"emblace" looked like a typo, and I could only find "emplace_back" in a C++ search.